src/templates/substance.html

```diff
@@ -961,7 +1006,7 @@ <h2>Studies & References</h2>
             }
 
             // Process after-effects with gradual decline
-            if (shouldDisplayAfterEffects && timingData.aftereffects && (timingData.aftereffects[roa] || timingData.aftereffects.value)) {
+            if (DISPLAY_AFTER_EFFECTS && timingData.aftereffects && (timingData.aftereffects[roa] || timingData.aftereffects.value)) {
```

Comment:
Where is `DISPLAY_AFTER_EFFECTS` modified? Is it just there as a WIP to disable the gradual decline animation until this feature is finished?